flickit-assessment-kit/src/main/java/org/flickit/assessment/kit/application/port/out/question/LoadQuestionsPort.java

```diff
@@ -7,4 +7,11 @@
 public interface LoadQuestionsPort {
 
     List<Question> loadAllByKitVersionId(long kitVersionId);
+
+    List<Result> loadQuestionsWithoutAnswerRange (long kitVersionId);
```

Comment:
Run reformatting on this file.
Before an opening parenthesis, there should be no space.